src/autoload/Multiplayer.gd

```diff
@@ -127,16 +181,19 @@ remote func register_player(name: String):
 # Disconnect from the session.
 func disconnect_from_session() -> void:
 	Input.mouse_mode = Input.MOUSE_MODE_VISIBLE
-	get_tree().network_peer = null
+	_cleanup_network_peer()
 	player_info = {}
-	var error := get_tree().change_scene("res://src/states/Menu.tscn")
-	assert(not error)
+	if dedicated_server:
```

Comment:
Can this even be called from the dedicated server? Isn't this a button action

Reply:
If someone is running on a desktop platform with a graphical environment, they can still press buttons and such. This is to make those things still work without issues. Someone running using the `Server` platform won't be able to use these ofc